app/src/main/res/raw/webviewcompat_complex_test_script.js

```diff
@@ -0,0 +1,2248 @@
+(function() {
```

Comment:
Is this test script just large, or is it large and also _doing stuff_ ? I see things like `PerformanceMonitor.measure` etc... but unclear if that is just a lot of stuff to test out the large script hypothesis or if this will result in things also being executed (which would be a confounding variable perhaps).